neqo-transport/tests/retry.rs

```diff
@@ -37,12 +37,10 @@ fn retry_basic() {
 
     let dgram = client.process_output(now()).dgram(); // Initial
     assert!(dgram.is_some());
```

Comment:
Just confirming that we *don't* use the default IP TOS for a typical Initial, because we set ECT(0).  In that case, this is good.

Reply:
Correct.